pkg/kando/location_delete.go

```diff
@@ -32,43 +29,20 @@ func newLocationDeleteCommand() *cobra.Command {
 		Short: "Delete artifacts from s3-compliant object storage",
 		// TODO: Example invocations
 		RunE: func(c *cobra.Command, args []string) error {
-			return runLocationDelete(c)
+			if err := validateCommandArgs(c); err != nil {
+				return err
+			}
+			dataMover, err := dataMoverFromCMD(c, kopiaSnapshotFlagName)
```

Comment:
When we call this function, we already know the type if `flag` we pass as a second argument. In the function definition, we are again calling `getKopiaSnapshotAndOutputNameFlag()` which has switch case. Can get rid of the function as we can directly get the value from here?

Reply:
Implemented